zebra-chain/src/orchard/shielded_data.rs

```diff
@@ -51,6 +55,18 @@ impl AuthorizedAction {
     }
 }
 
+impl ZcashDeserialize for AuthorizedAction {
```

Comment:
Please delete this implementation - the spec says that actions and their signatures are serialised in separate arrays:
![1C61D58D-F94F-4DE9-A3DE-4F36C8362380](https://user-images.githubusercontent.com/8951843/118185993-8976a600-b480-11eb-817e-42674917ee83.jpeg)

And the design says:

> The following types have ZcashSerialize and ZcashDeserialize implementations, because they can be serialized into a single byte vector:
> * Action (new)
> * redpallas::Signature<SpendAuth> (new)
> 
> Note: AuthorizedAction does not have serialize and deserialize implementations. It must be split using into_parts before serialization, and recombined using from_parts after deserialization.

https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/rfcs/0010-v5-transaction.md#adding-orchard-authorizedaction

Reply:
Do'h. Thanks. https://github.com/ZcashFoundation/zebra/pull/2116/commits/96d84d8461e55401427c46cb6d38c96e5d25cd00